### PR TITLE
Fix show

### PR DESCRIPTION
**Pull request by saschatimme**
First commit is a fix of the fix #96, sorry that this is necessary in the first place...
Second commit makes it more probable that our printing can be parsed by other programs.

### CODE REVIEW

File: src/show.jl [hunk 1]
```diff
@@ -85,6 +85,9 @@ function print_term(io::IO, mime, t::AbstractTerm)
                 print(io, '-')
             else
                 print_coefficient(io, coefficient(t))
+                if !iszero(t)
+                    print_maybe_mul(io, mime)
```
Comment (blegat) [inline src/show.jl]: Could add a comment for why this is added

File: src/show.jl [hunk 2]
```diff
@@ -93,6 +96,8 @@ function print_term(io::IO, mime, t::AbstractTerm)
     end
 end
 
+print_maybe_mul(io::IO, ::MIME"text/print") = print(io, "*")
```
Comment (blegat) [inline src/show.jl]: Or maybe add the comment here

**Comment by saschatimme** [comment]
Just added a comment as well as an doc string.

File: src/show.jl [hunk 3]
```diff
@@ -86,7 +86,9 @@ function print_term(io::IO, mime, t::AbstractTerm)
             else
                 print_coefficient(io, coefficient(t))
                 if !iszero(t)
-                    print_maybe_mul(io, mime)
+                    # Print the a multiplication sign between coefficent and monmomial
```
Comment (blegat) [inline src/show.jl]: "the a"

**Pull request merged**